config/log4j2.properties

```diff
@@ -0,0 +1,36 @@
+status = error
+name = LogstashPropertiesConfig
+
+property.filename = /tmp/logstash/logstash.log
```

Comment:
OK, if elasticsearch will do so, I surrender :)
I'm however surprised by this choice, since properties format was firstly abandoned by log4j2 and has been finally added in log4j 2.4, using a more complex format than log4j 1.x

Anyway, concerning support for another log4j2 config file, it requires just 2 lines of code :)